src/views/SignUpView.vue

```diff
@@ -127,8 +127,8 @@ export default {
         };
         try {
           await authService.actions.signup(user);
+          this.$router.push(`/profile/${this.email}`);
```

Comment:
not sure how I feel about using emails in the URL, i feel like that won't look pretty

Reply:
Agreed!  But we don't have any other info from the backend about the user. I think were working on that next sprint.